snow/engine/snowman/syncer/state_syncer.go

```diff
@@ -234,6 +234,11 @@ func (ss *stateSyncer) AcceptedStateSummary(ctx context.Context, nodeID ids.Node
 	ss.pendingVoters.Remove(nodeID)
 
 	weight := ss.StateSyncBeacons.GetWeight(nodeID)
+	ss.Ctx.Log.Debug("adding weight to summaries",
+		zap.Stringer("nodeID", nodeID),
+		zap.Stringers("summaryIDs", summaryIDs),
+		zap.Uint64("weight", weight),
```

Comment:
nit: should we use `currentWeight` instead of just `weight`. Same for log post increment

Reply:
I did a pass to standardize + clarify the weights being logged here.